ebk_client.py

```diff
@@ -145,61 +154,41 @@ def delete_ad(self, ad_id):
+            url = f'/categories/{cat_id}.json'
         else:
-            url = "/categories.json"
-            schema_uri = '{http://www.ebayclassifiedsgroup.com/schema/category/v1}categories'
-
-        response = self._http_get(url)
-        response_json = json.loads(response.content.decode('utf-8'))
-        categories = response_json.get(schema_uri, {}).get('value', {}).get('category', None)
+            url = '/categories.json'
+        return self.__http_get_json_content(url).get('category', None)
 
-        return categories
+    def get_category_attributes(self, cat_id=None):
+        url = f'/attributes/metadata/{cat_id}.json'
+        return self.__http_get_json_content(url).get('attribute', None)
 
-    def get_category_attributes(self, cat_id = None):
+    def get_category_metadata(self, cat_id=None):
+        url = f'/ads/metadata/{cat_id}.json'
+        return self.__http_get_json_content(url)
 
-        url = "/attributes/metadata/{}.json".format(cat_id)
-        schema_uri = '{http://www.ebayclassifiedsgroup.com/schema/attribute/v1}attributes'
-
-        response = self._http_get(url)
-        response_json = json.loads(response.content.decode('utf-8'))
-        cat_attr = response_json.get(schema_uri, {}).get('value', {}).get('attribute', None)
-
-        return cat_attr
-
-    def get_locations(self, url_suffix, depth = None, include_parent_path = False):
-        url = "/locations.json?{}".format(url_suffix)
+    def get_locations(self, url_suffix, depth=None, include_parent_path=False):
+        url = f'/locations.json?{url_suffix}'
         if depth is not None:
-            url += '&depth={}'.format(depth)
+            url += f'&depth={depth}'
 
         if include_parent_path is True:
             url += '&includeParentPath=true'
 
-        schema_uri = '{http://www.ebayclassifiedsgroup.com/schema/location/v1}locations'
-
-        response = self._http_get(url)
-        response_json = json.loads(response.content.decode('utf-8'))
-        categories = response_json.get(schema_uri, {}).get('value', {}).get('location', None)
-
-        return categories
-
-    def get_location_by_name(self, location_name, depth = None, include_parent_path = False):
-        url_suffix = "q={}".format(location_name)
-        locations = self.get_locations(url_suffix, depth, include_parent_path)
-        return locations
+        return self.__http_get_json_content(url).get('location', None)
 
-    def get_location_by_coordinates(self, latitude, longitude, depth = None, include_parent_path = False):
-        url_suffix = "latitude={}&longitude={}".format(latitude,longitude)
-        locations = self.get_locations(url_suffix, depth, include_parent_path)
-        return locations
+    def get_location_by_name(self, location_name, depth=None, include_parent_path=False):
+        url_suffix = f'q={location_name}'
+        return self.get_locations(url_suffix, depth, include_parent_path)
 
-    def html_unescape(self, data):

```

Comment:
As you removed this method, can you please fix example.py (line 47)?

Reply:
Sorry. I missed that.
Due to minimum python version of 3.6 the whole try-import cascade is not necessary anymore.